cassdegrees/templates/createlist.html

```diff
@@ -0,0 +1,91 @@
+{% extends "base.html" %}
+{% load static %}
+{% load breadcrumbs %}
+
+{% block page-title %}{% if edit %}Edit{% else %}Create{% endif %} List{% endblock %}
+
+{% block subtitle %}{% if edit %}Edit{% else %}Create{% endif %} List{% endblock %}
+
+{% block breadcrumb-trail %}
+    {% breadcrumb '/admin/home' 'Admin Home'%}
+    {% breadcrumb '/admin/list/?view=Program' 'Programs/Subplans/Courses' %}
+    {% if edit %}
+        {% finalcrumb 'Edit List' %}
+    {% else %}
+        {% finalcrumb 'Create List' %}
+    {% endif %}
+{% endblock %}
+
+{% block content %}
+
+    <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+    <script src="https://unpkg.com/vue-multiselect@2.1.0"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vue-resource@1.5.1"></script>
```

Comment:
scripts typically go at the end. These should also be bundled with the application to minimise cross domain requests - e.g. we are already shipping vue.

Reply:
Have bundled the scripts, however the scripts must be declared before the first widget listing or the console will throw a "vue not defined" error.